shared/src/main/java/uk/gov/di/authentication/shared/entity/AuthSessionItem.java

```diff
@@ -23,10 +24,17 @@ public enum AccountState {
         UNKNOWN
     }
 
+    public enum ResetPasswordState {
+        NOT_APPLICABLE,
```

Comment:
Nitpick and happy for this to be left as-is, but perhaps just `NONE`?

Reply:
No preference from me, Done